config/tokenwrapper.json

```diff
@@ -0,0 +1,15 @@
+{
+    "mainnet": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    "polygon": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
+    "avalanche": "0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7",
+    "arbitrum": "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1",
+    "base": "0x4200000000000000000000000000000000000006",
+    "boba": "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000",
+    "bsc": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
+    "fantom": "0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83",
+    "gnosis": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    "linea": "0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f",
+    "optimism": "0x4200000000000000000000000000000000000006",
+    "polygonzkevm": "0x4F9A0e7FD2Bf6067db6994CF12E4495Df938E6e9",
+    "zksync": "0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91",
+}
```

Comment:
This list looks incomplete compared to the amount of chains we support. We should be able to extract the list from `wrappedTokens` in https://github.com/lifinance/data-types/blob/main/src/coins/coins.ts#L1730